### DISCUSSION TITLE
Build form Source Error

### DISCUSSION THREAD

**Discussion by pmaine** [AlexandreRouma/SDRPlusPlus#280] [Q&A] [answered]
I am building from source on a Raspberry Pi 4B and I get the following error:
cmake ..
-- Checking for module 'libairspy'
--   No package 'libairspy' found
CMake Error at /usr/share/cmake-3.16/Modules/FindPkgConfig.cmake:493 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake-3.16/Modules/FindPkgConfig.cmake:677 (_pkg_check_modules_internal)
  airspy_source/CMakeLists.txt:28 (pkg_check_modules)

-- Configuring incomplete, errors occurred!
See also "/home/pi/source/SDRPlusPlus/build/CMakeFiles/CMakeOutput.log".

**Comment by the-gobleo-tinkerer** [comment, marked as answer]
> `-- No package 'libairspy' found`

You don't have the Airspy library installed, which is required for Airspy R2 and Airspy Mini support. If you use an Airspy SDR device, run `sudo apt install libairspy-dev` to install the files needed; if you don't need Airspy support, change `OPT_BUILD_AIRSPY_SOURCE` to `OFF` in CMakeLists.txt and try compiling again.

**Reply by pmaine** [reply]
The above response answered my question but GitHub wont let me mark as answered...
